code/traditional/TCA/TCA.py

```diff
@@ -61,13 +65,20 @@ def fit(self, Xs, Xt):
         a, b = np.linalg.multi_dot([K, M, K.T]) + self.lamb * np.eye(n_eye), np.linalg.multi_dot([K, H, K.T])
         w, V = scipy.linalg.eig(a, b)
         ind = np.argsort(w)
+        
+        # Projecting to latent space
         A = V[:, ind[:self.dim]]
         Z = np.dot(A.T, K)
         Z /= np.linalg.norm(Z, axis=0)
-        Xs_new, Xt_new = Z[:, :ns].T, Z[:, ns:].T
-        return Xs_new, Xt_new
+        
+        Xs_new, Xt1_new = Z[:, :ns].T, Z[:, ns:].T
+        
+        self.W_proj = A
```

Comment:
Maybe it is enough to only retain the W_proj?

Reply:
sorry don't get it, let me confirm the W_proj is A right? so if you say not to change the .fit method, I should create another method again that build W_proj and/or projects it into Xt2, like 
1. `fit_new(Xs, Xt1, Xt2) -> Xt2_new, W_proj`
or separate them?
1. `fit_wproj (Xs, Xt1) -> W_proj`
2. `fit_new(Xt2) -> Xt2_new`

Reply:
Yes. My key point is not to modify the existing TCA functions (interfaces) but try to add a new function for new target data. That's all.

Reply:
all right, working on it.